conf/messages

```diff
@@ -1565,3 +1617,12 @@ businessRatesAttachment.file.size.reached.max.limit=You have uploaded 1 file, th
 fileupload.support.documents.size.reached.max.limit=You have uploaded 1 file, the maximum allowable limit
 fileupload.pleaseWait=Please wait whilst your file is uploading. This may take some time.
 
+#Javascript Error messages
+label.selectValue=Select an address,
+addressHelp=This should be your preferred postal contact address for your business.,
+label.loading=Loading...,
+label.showing=Showing,
+label.to=to,
+label.of=of,
+label.submissionId=Submission ID,
```

Comment:
Not sure these are used, but if they are I think the **trailing comma** is wrong.

Looks like they are copied from the old messages.js file.

Reply:
I have cleaned up lots of messages fron old messages file and removed unused JS file
I have checked again and I have remove others but keep the following as been used 
label.selectValue=Select an address,
addressHelp